### ISSUE TITLE
[Bug Report] Hopper "global" XML error

### ISSUE THREAD

**Issue by rickstaa** [Farama-Foundation/Gymnasium#597]
### Describe the bug

When trying to run the [Hopper-v4](https://github.com/Farama-Foundation/Gymnasium/blob/main/gymnasium/envs/mujoco/hopper_v4.py) environment, I receive the following error:

```bash
ValueError: XML Error: global coordinates no longer supported. To convert existing models, load and save them in MuJoCo 2.3.3 or older
Element 'compiler', line 2
```

As explained in https://github.com/deepmind/mujoco/issues/833, this is caused by an upstream change in the [Mujoco library](https://github.com/deepmind/mujoco). For now, the best way to solve this is to pin Mujoco to version `2.3.3`. I, however, wanted to open this issue to track the release of https://github.com/Farama-Foundation/Gymnasium/pull/589. Feel free to close this issue if you think this is unnecessary.

### Code example

1. Install Mujoco `v0.28.0`.
2. Try to run the following code:

```python
import gymnasium as gym

if __name__ == "__main__":
    env = gym.make('Hopper-v4', render_mode='human')

    env.reset()
    print("Testing Hopper-v4 environment.")
    for _ in range(1000):
        env.render()
        env.step(env.action_space.sample())

    env.close()
    print("Test finished.")
```

3. Be greeted by the error.

### System info

Gymnasium install type: `pip`
Pip version: 23.1.2
Gymnasium version: 0.28.1
Mujoco version: 2.3.6
OS: Ubuntu 20.04
Python: 3.8.10

### Additional context

Will be solved when https://github.com/Farama-Foundation/Gymnasium/pull/589  is released.

### Checklist

- [X] I have checked that there is no similar [issue](https://github.com/Farama-Foundation/Gymnasium/issues) in the repo

**Comment by rickstaa** [comment]
Maybe this is already tracked in https://github.com/Farama-Foundation/Gymnasium/pull/572?

**Comment by pseudo-rnd-thoughts** [comment]
This has already been fixed in https://github.com/Farama-Foundation/Gymnasium/pull/589
We will hopefully have a release soon for everyone to use

**Comment by Longyouyuan** [comment]
> This has already been fixed in #589 We will hopefully have a release soon for everyone to use

but how to use the file to fix the problem?

**Comment by pseudo-rnd-thoughts** [comment]
> > This has already been fixed in #589 We will hopefully have a release soon for everyone to use
> 
> 
> 
> but how to use the file to fix the problem?

You should just need to use gymnasium v0.29.0
